src/en/guide/gettingStarted/supportedJavaEEContainers.adoc

```diff
@@ -1,15 +1,3 @@
-Grails runs on any container that supports Servlet 3.0 and above and is known to work on the following specific container products:
+The Grails framework requires that runtime containers support Servlet 3.0 and above. By default, Grails framework applications are bundled with an embeddable Tomcat.  For more information, please see the "Deployment" section of this documentation.
```

Comment:
I was going to suggest that "see the Deployment section" be changed to an actual link.

Then noticed that https://docs.grails.org/latest/guide/deployment.html (specifically, "Application servers" in section 21.2) actually refers back to this section indicating that we have a list of servers that Grails has been tested with, so we might want to change that as well.

Otherwise, all things good.

Reply:
I updated that section as well